lib/runcible/base.rb

```diff
@@ -82,6 +79,9 @@ def call(method, path, options = {})
       response = get_response(client, path, *args)
       process_response(response)
 
+    rescue RestClient::ResourceNotFound => e
+      log_exception(404)
```

Comment:
Given we have a paradigm of defining log type methods, e.g. `log_exception`, `log_debug`, I think a `log_info` would make sense instead of overloading the `log_exception` method.